tests/components/device_automation/test_init.py

```diff
@@ -498,43 +520,54 @@ def _same_lists(a, b):
         {
             automation.DOMAIN: {
                 "alias": "hello",
-                "trigger": {"platform": "event", "event_type": "test_event1"},
-                "action": {"device_id": "", "domain": "test"},
+                "trigger": {
+                    "platform": "event",
+                    "event_type": "test_event1"
+                },
+                "action": {
+                    "device_id": "",
+                    "domain": "test"
+                },
             }
         },
     )
 
-    assert (
-        "Integration 'test' does not support device automation actions" in caplog.text
-    )
+    assert ("Integration 'test' does not support device automation actions" in
+            caplog.text)
 
 
-async def test_automation_with_integration_without_device_condition(hass, caplog):
+async def test_automation_with_integration_without_device_condition(
+        hass, caplog):
     """Test automation with integration without device condition support."""
     assert await async_setup_component(
         hass,
         automation.DOMAIN,
         {
             automation.DOMAIN: {
                 "alias": "hello",
-                "trigger": {"platform": "event", "event_type": "test_event1"},
+                "trigger": {
+                    "platform": "event",
+                    "event_type": "test_event1"
+                },
                 "condition": {
                     "condition": "device",
                     "device_id": "none",
                     "domain": "test",
                 },
-                "action": {"service": "test.automation", "entity_id": "hello.world"},
+                "action": {
+                    "service": "test.automation",
+                    "entity_id": "hello.world"
+                },
             }
         },
     )
 
-    assert (
-        "Integration 'test' does not support device automation conditions"
-        in caplog.text
-    )
+    assert ("Integration 'test' does not support device automation conditions"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4564454)